admin/app/controllers/RadiatorController.scala

```diff
@@ -26,7 +26,7 @@ object RadiatorController extends Controller with Logging with AuthLogging with
 
   def switchesExpiringThisWeek = {
     Switches.all.filter { switch =>
-      switch.sellByDate.isBefore(new LocalDate().plusDays(7))
+      Switch.expiry(switch).expiresSoon
```

Comment:
Perhaps `expiresSoon` should be `expiresThisWeek` if the two concepts are coupled together.